apps/api/src/config/env-validator.ts

```diff
@@ -44,6 +54,8 @@ const validators: { [K in keyof any]: ValidatorSpec<any[K]> } = {
   NEW_RELIC_LICENSE_KEY: str({
     default: '',
   }),
+  CACHE_HOST: recommended({ default: '' }),
+  CACHE_PORT: recommended({ default: '' }),
```

Comment:
```suggestion
  REDIS_CACHE_HOST: str({ default: '' }),
  REDIS_CACHE_PORT: str({ default: '' }),
```

Would remove the recommended flag, but would suggest to add a `REDIS` prefix, just to make it more clear. Wdyt?

Reply:
So you suggest adding the env variable as mandatory? because I really was not sure we should force the users to cache if they did not mean to.
Let me know what you think. @scopsy 
Regarding the name i will update it, for me it is always harder to use less generic names 😅

Reply:
@scopsy